website/docs/r/host.html.markdown

```diff
@@ -103,6 +109,11 @@ The following arguments are supported:
 ~> **NOTE:** Tagging support is not supported on direct ESXi host
 connections and require vCenter Server.
 
+* `services` - (Optional) Set Services on host, the settings to be set are based on service being set as part of import.
```

Comment:
Theres a make routine that generates docs. You should run that so that this doesn't get wiped the next time someone regenerates this MD file

Reply:
can you show me how to do this?

Reply:
my bad, this project doesn't have that.

Reply:
yep, just in the vcf provider. we have a backlog item for that here.